scripts/data/src/org/akvo/gae/remoteapi/ExportAuthDataToEventLog.java

```diff
@@ -38,11 +42,13 @@ public void execute(DatastoreService ds, String[] args) throws Exception {
                     System.out.println("progress " + amount);
                 }
                 Map<String, Object> event = createEvent(entity, eventTypes, orgId);
-                Date timestamp = (Date) entity.getProperty(EventUtils.Prop.LAST_UPDATE_DATE_TIME);
+                Date timestamp = new Date();
                 Entity eventLogEntity = EventUtils.createEventLogEntity(event, timestamp);
                 ds.put(eventLogEntity);
             }
 
+            System.out.println("amount = " + amount);
```

Comment:
Was this for personal debugging?  We usually use the Logger if we want this in the GAE logs.

Reply:
This code is not going to be run in GAE but in the local dev box.

The information is useful to know how big the dataset was exported.